.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,53 @@
+name: Deploy as Google function
+
+on:
+  push:
+    branches:
+      -  main
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.12
+
+      - name: Test
+        run: go test -v ./...
+
+      - name: Auth gcloud CLI for deploy
+        uses: google-github-actions/auth@main
+        with:
+          credentials_json: ${{ secrets.DIGDIR_CLOUD_FUNCTIONS_SA_KEY }}
+
+      - name: Deploy
+        uses: google-github-actions/deploy-cloud-functions@main
+        with:
+          name: user-feedback-service
+          runtime: go116
+          entry_point: EntryPoint
+          memory_mb: 256
+          region: europe-west1
+          max_instances: 1
+          env_vars: ${{ secrets.ENV_VARS }}
+
```

Comment:
overflødig linjeskift

Reply:
Er det ønskelig å fjerne alle tomme linjer her?

Reply:
Det er ønskelig med én tom linje mellom hver step-blokk. Akkurat her var det to